examples/specs/concat_layer_voyager_result_future.vl.json

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
Why not cut the whole example and include the lasagna plot instead?

Reply:
They are totally different customization.  This one is customizing grid.

Reply:
Okay, but let's remove the labels to the right. Also, why is the title cut off?

Reply:
Ok Fixed.  Note that the example is still long because we don't have `align` channel yet (#3726).

Title is only cut off for SVG-text, but not on editor.